disassemblers/ofrak_angr/ofrak_angr_test/test_unpackers.py

```diff
@@ -123,3 +132,46 @@ def _split_bb(
 
         expected_results[cb_addr][idx] = bb_1
         expected_results[cb_addr].append(bb_2)
+
+
+async def test_basic_block_no_exit(ofrak_context: OFRAKContext):
+    # Regression test for https://github.com/redballoonsecurity/ofrak/issues/614
+    # Test unpacking a ComplexBlock that contains a BasicBlock which doesn't have an exit address
+    with tempfile.TemporaryDirectory() as temp_dir:
+        response = requests.get(
+            "https://busybox.net/downloads/binaries/1.35.0-x86_64-linux-musl/busybox"
+        )
+        response.raise_for_status()
```

Comment:
We should download this, add it to one of the test `assets` directories, and make sure it is LFS-committed.